specificatie/openapi.yaml

```diff
@@ -221,11 +221,11 @@ components:
           type: string
           title: WOZ-objectnummer
           description: Unieke identificatie van het WOZ-object
-        kadastraalOnroerendeZaken:
+        kadastraalOnroerendeZaakIdentificaties:
           type: array
           description: De kadastraal onroerende zaken die geheel of gedeeltelijk deel uitmaken van het WOZ-object
```

Comment:
Misschien in de description aangeven dat het de unieke identificatie van de Kadastraal Onroerende zaak betreft zoals hierboven bij de adresseerbaarObjecten ook gedaan is. (Als je dat doet, ook doorvoeren in de genereervarianten)